toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -2159,6 +2170,33 @@ func runPostInstallScripts(installChroot *safechroot.Chroot, config configuratio
 	return
 }
 
+func cleanupTdnfCache(installChroot *safechroot.Chroot) error {
+	const (
+		squashErrors      = false
+		rpmCacheDirectory = "/var/cache/tdnf"
+	)
+
+	ReportActionf("Cleaning tdnf cache")
+	err := installChroot.UnsafeRun(func() error {
+		chrootErr := shell.ExecuteLive(squashErrors, "tdnf", "clean", "all")
```

Comment:
Let me know if this line works. When I tried the same I was running into some error, I assumed I needed to provide `--installroot` parameter

Reply:
Might be running from the setup chroot rather than the install chroot, that would likely require --installroot.